src/container/routes/constants.js

```diff
@@ -0,0 +1,3 @@
+export const MAIN_ROUTE = '/';
```

Comment:
should b a pascal file name.
Push your code after finalizing stuff

Reply:
I did convert it into Pascal case all the file but its not updating on github just test it on test commit , github ignores the capitalization of filename (just asked from Shazil).

Reply:
constants.js is not the JSX, I asked from Ahmed about the naming convention of js files,as he was suggested to name it like that (constants.js) thats why I also name js files in the data file camels cases, from now on I'll go with Pascal